src/main.c

```diff
@@ -387,10 +390,11 @@ int main(int argc, char* args[])
     {"console", 'c', OPTPARSE_NONE},
     #endif
     {"debug", 'd', OPTPARSE_NONE},
+    {"embed", 'e', OPTPARSE_REQUIRED},
```

Comment:
This is not sufficient. Because optparse _expects_ this to be followed by an argument, different forms of invocation can break.

For example:
```bash
./dome module.wren -e    # This will error out.
./dome module.wren -e module  # This seg faults because it tries to use "module" as a file name.
```

This should be `OPTPARSE_NONE`, and we should do the argument checks ourselves (including the actual presence of the input file.)

Reply:
ok